docs/development/new-packages.md

```diff
@@ -150,12 +150,13 @@ and see if there are any errors.
 
 If the build succeeds you can try to load the package:
 
-1. Serve the dist directory with `python -m http.server --directory ./dist`.
+1. Build pyodide and all associated packages via `make all`.
+2. Serve the dist directory with `python -m http.server --directory ./dist`.
    If you use docker, you can execute this either outside of the docker container or
    make sure to forward a port by setting the environment variable
    PYODIDE_SYSTEM_PORT or starting docker with `./run_docker -p <port>`.
-2. Open `localhost:<port>/console.html` and try to import the package.
-3. You can test the package in the repl.
+3. Open `localhost:<port>/console.html` and try to import the package.
```

Comment:
If you used `python -m http.server` it will have selected port 8000, so maybe we should just put that explicitly here? Not sure.

Reply:
Yes, I think that makes sense. Anyone who intentionally modifies the port would know to change it here too

```suggestion
3. Open `localhost:8000/console.html` and try to import the package.
```